### PR TITLE
steering: Update membership following 2021 election

### PR DESCRIPTION
**Pull request by justaugustus**
Part of https://github.com/kubernetes/steering/issues/219.

Emeritus:
- @nikhita 
- @derekwaynecarr 

Returning:
- @parispittman 
- @cblecker 

New:
- @justaugustus 
- @tpepper 

Includes a sync of the OWNERS_ALIASES with k/community, which brings in the new Steering members (as well as additional governance group updates).

Signed-off-by: Stephen Augustus <foo@auggie.dev>

/assign @cblecker @mrbobbytables 
cc: @kubernetes/steering-committee

### CODE REVIEW

**Review by idvoretskyi — APPROVED** [review]
/lgtm

/hold

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -21,8 +21,8 @@ aliases:
     - gjtempleton
     - mwielgus
   sig-cli-leads:
-    - eddiezane
     - KnVerey
+    - eddiezane
```
Comment (ameukam) [inline OWNERS_ALIASES:25]: Is this change needed ?

Comment (liggitt) [reply OWNERS_ALIASES:25]: it's easier to sync the whole OWNERS_ALIASES file from k/k than try to pull in manual updates

Comment (justaugustus) [reply OWNERS_ALIASES:25]: @ameukam -- This was copied in from k/community's generator, which is sorting ASCIIbetically (so uppercase usernames are sorting in first).
